README.md

```diff
@@ -1,62 +1,64 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
-
```

Comment:
- [OPTIONAL ] You should add description to the project. You can easily get project information from the activity in your Dashboard.

Reply:
ok thanks